### ISSUE TITLE
Proposal: Shell —— Reorder markup slightly to improve accessibility

### ISSUE THREAD

**Issue by ShayMurnin** [carbon-design-system/ibm-security#895]
## Proposal - 
I've been going through some accessibility violations on Falcon that are being flagged by the `accessibility-checker` browser extension and noticed a violation being flagged for the `Shell` component when the `← return to banner` is shown.

Violation: **2.4.1 Bypass Blocks**
> 🚫 Content is not within a landmark element

My proposal is to satisfy this accessibility rule by rearranging the structure of the markup slightly.

#### As-is

```
<a /> // return to banner
<div>
	<div>
		<nav /> // navigation toolbar
	</div>
	<div>
		<header /> // security header
	</div>
</div>
```

#### Proposed to-be

```
<header>
	<a /> // return to banner
	<div>
		<nav /> // navigation toolbar
	</div>
	<div>
		<div />  // security header
	</div>
</header>
```

With these changes, the rule passes. (see below — *some styling changes also needed for the left margin and another a11y violation on the profile button)

> Left, as-is. RIght, to-be.
![image](https://user-images.githubusercontent.com/15728451/105858162-ebe78b80-5fe2-11eb-8b2f-5c363da22bc8.png)

Does this work semantically in terms of the HTML? That's my concern. Let me know your thoughts. I can follow up with a PR. 

✌️

### Acceptance criteria

- [ ] `← Return to banner` does not violate accessibility rules on `accessibility-checker`
- [ ] HTML is semantically correct

**Comment by SimonFinney** [comment]
Hey @ShayMurnin - Thanks for raising! I've been thinking about this - Would it be sufficient to add `role="navigation"` to the banner itself?

**Comment by Jasonaddley** [comment]
Hi @SimonFinney I think we need the `<header>` structure to pass this rule.

**Comment by SimonFinney** [comment]
In this case, it might be sufficient to convert the `Shell`'s containing element to `header`

**Comment by joemc11** [comment]
Still an issue
